torch_xla/core/xla_model.py

```diff
@@ -860,7 +860,31 @@ def get_stablehlo(tensors=None):
   if tensors is None:
     tensors = []
   return torch_xla._XLAC._get_stablehlo(
-      tensors, torch_xla._XLAC._xla_get_default_device(), [])
+      tensors, torch_xla._XLAC._xla_get_default_device(), [],
+      False).decode('utf-8')
+
+
+def get_stablehlo_bytecode(tensors=None) -> bytes:
```

Comment:
Thanks! can we add the new api to https://github.com/pytorch/xla/blob/master/docs/source/index.rst#xla_model ?